app/views/press_catalog/_index_header_list_monograph.html.erb

```diff
@@ -30,19 +30,25 @@
 
   <% if presenter.epub? %>
     <div class="row epub">
-      <div class="col-sm-12" data-turbolinks="false">
-        <a class="btn btn-default btn-lg" href="<%= epub_path(presenter.epub, publisher: presenter.subdomain) %>" title="Read <%= presenter.title %>" role="button">Read</a>
+      <div class="col-sm-12">
+        <form action="<%= epub_path(presenter.epub, publisher: presenter.subdomain) %>">
+          <button type="submit" class="btn btn-default btn-lg">Read</button>
+        </form>
       </div>
     </div>
-    <div class="row info" data-turbolinks="false">
+    <div class="row info">
       <div class="col-sm-12">
-        <a class="btn btn-default" href="<%= polymorphic_path(url_for_document(document)) %>">More Info</a>
```

Comment:
Keep the original markup as an `<a ... >` but remove the `role="button"` attribute from the tag.